src/actions.jl

```diff
@@ -71,5 +89,22 @@ function jet(str, should_print=false)
     else
       error("Unexpected action $kind. See Jet's help")
     end
+  # Currently supported searches with more than a single ':' come in the form of `pkg:Package cmd:Command`
+  elseif length(s) == 3
+    first = findfirst(':', str)
+    last = findlast(':', str)
+    sep = findlast(' ', str[1:last])
+
+    kind = [str[1:first-1], str[sep+1:last-1]]
+    pkg, keyword = [str[first+1:sep-1], str[last+1:end]]
+
+    if kind[1] in ["pkg", "package"] &&
+       kind[2] in ["cmd", "command", "snippet"]
+      jet_snippet_in(keyword, pkg, should_print)
```

Comment:
```suggestion
    kws = [str[first+1:sep-1], str[last+1:end]]

    if kind[1] in ["pkg", "package"] &&
       kind[2] in ["cmd", "command", "snippet"]
      jet_snippet_in(keyword, kws[1], kws[2])
    elseif kind[2] in ["pkg", "package"] &&
           kind[1] in ["cmd", "command", "snippet"]
      jet_snippet_in(keyword, kws[2], kws[1])
```
This was we can accept `pkg:... cmd:...` and `cmd:... pkg:...` as well.